src/Peachpie.CodeAnalysis/Syntax/PhpSyntaxTree.cs

```diff
@@ -206,6 +206,21 @@ public override FileLinePositionSpan GetLineSpan(TextSpan span, CancellationToke
             return new FileLinePositionSpan(_source.FilePath, _source.LinePosition(span.Start), _source.LinePosition(span.End));
         }
 
+        /// <summary>
+        /// Returns the offset of the location specified by (zero-based) line and character from the start of the file.
+        /// In the case of invalid line, -1 is returned.
+        /// </summary>
+        public int GetPosition(LinePosition linePosition)
```

Comment:
We can, but it is used from the language server, so it must be public. Should I make AstUtils public, then?